Lib/test/test_pyrepl/test_pyrepl.py

```diff
@@ -862,6 +862,29 @@ def test_dumb_terminal_exits_cleanly(self):
         self.assertNotIn("Exception", output)
         self.assertNotIn("Traceback", output)
 
+    @force_not_colorized
+    def test_python_basic_repl(self):
+        env = os.environ.copy()
+        commands = ("from test.support import initialized_with_pyrepl\n"
+                    "initialized_with_pyrepl()\n"
+                    "exit()\n")
+
+        env.pop("PYTHON_BASIC_REPL", None)
+        output, exit_code = self.run_repl(commands, env=env)
+        if "can\'t use pyrepl" in output:
+            self.skipTest("pyrepl not available")
+        self.assertEqual(exit_code, 0)
+        self.assertIn("True", output)
```

Comment:
(To double-check that the test is working as expected)

```suggestion
        self.assertIn("True", output)
        self.assertNotIn("False", output)
```